src/checkpoint/checkpoint_txn.c

```diff
@@ -1494,8 +1494,19 @@ __checkpoint_db_internal(WT_SESSION_IMPL *session, const char *cfg[])
      * view of the data, make sure that all the logs are flushed to disk before the checkpoint is
      * complete.
      */
-    if (F_ISSET(&conn->log_mgr, WT_LOG_ENABLED))
+    if (F_ISSET(&conn->log_mgr, WT_LOG_ENABLED)) {
+        /* Crash before metadata sync if checkpoint crash point is configured.
```

Comment:
Add this as part of the FIXME comment.

Reply:
fixed 22a4cd9f225f2a91e65e4f662a3ca4c16798fa26